plugins/scaffolder-backend/src/scaffolder/stages/helpers.ts

```diff
@@ -40,7 +38,7 @@ export const parseLocationAnnotation = (
   // split on the first colon for the protocol and the rest after the first split
   // is the location.
   const [protocol, location] = annotation.split(/:(.+)/) as [
-    RemoteProtocol?,
+    ('file' | 'url')?,
```

Comment:
If you are strict about this type (not just making it `string`) for some reason, you should really have a check and throw an error if it turned out to not be one of those two values. Or just make it `string`